core/src/main/java/org/elasticsearch/action/search/TransportMultiSearchAction.java

```diff
@@ -52,19 +55,23 @@ public TransportMultiSearchAction(Settings settings, ThreadPool threadPool, Tran
         this.clusterService = clusterService;
         this.searchAction = searchAction;
         this.availableProcessors = EsExecutors.numberOfProcessors(settings);
+        relativeTimeProvider = System::nanoTime;
```

Comment:
Can we prefix this with `this`, it will make it symmetric with the other fields, and the other constructor.